src/thor/bidirectional_astar.cc

```diff
@@ -1106,12 +1109,11 @@ void BidirectionalAStar::SetDestination(GraphReader& graphreader,
       dist = astarheuristic_forward_.GetDistance(tile->get_node_ll(opp_dir_edge->endnode()));
     }
     edgelabels_reverse_.emplace_back(kInvalidLabel, opp_edge_id, edgeid, opp_dir_edge, cost, sortcost,
-                                     dist, mode_, c, !opp_dir_edge->not_thru(),
-                                     !(costing_->IsClosed(directededge, tile)),
+                                     dist, mode_, c, false, !(costing_->IsClosed(directededge, tile)),
+                                     !(directededge->destonly() ||
```

Comment:
when setting the destination however, we want to look at the passed directed edge (not the opposing), because we put the opposing in the queue